pyi.py

```diff
@@ -531,15 +532,10 @@ def _check_subscript_version_check(self, node: ast.Compare) -> None:
         version_info = node.left
         if isinstance(version_info, ast.Subscript):
             slc = version_info.slice
-            if isinstance(slc, (ast.Index, ast.Num)):
-                # Python 3.9 flattens the AST and removes Index, so simulate that here
-                slice_num = slc if isinstance(slc, ast.Num) else slc.value
+            # FIXME: ast.Num isn't used on new Python versions
```

Comment:
It is for now, actually. ast.Num still exists but it's a shim that overrides `__instancecheck__` or something crazy like that. On Python 3.10:

```
In [5]: isinstance(ast.Constant(value=3), ast.Num)
Out[5]: True

In [6]: type(ast.Constant(value=3))
Out[6]: ast.Constant
```

Reply:
Sometimes you really just want Python to Stop Changing

Reply:
```suggestion
            # TODO: ast.Num works, but is deprecated
```